### PR TITLE
Add itemProp for ContentURL to image

### PR DESCRIPTION
**Pull request by mblakedev**

### CODE REVIEW

**Comment by tadjohnston** [comment]
Is there an associated story? If so, can you please add it?

**Comment by mblakedev** [comment]
Story related to Rentals PDP apartment complex schema: https://rentpath.leankit.com/card/550423158

**Review by orionstein — APPROVED** [review]
Looks good!

File: package.json [hunk 1]
```diff
@@ -2,11 +2,12 @@
...
   "devDependencies": {
-    "coffee-script": "~1.6.0",
+    "coffee-script": "~1.7.0",
     "bower": "~1.2.8",
     "bower-requirejs": "~0.8.6"
   },
   "scripts": {
-    "start": "coffee -w -o . -c src"
+    "start": "coffee -w -o . -c src",
+    "build": "coffee -o . -c src"
```
Comment (tadjohnston) [inline package.json]: 👍

**Review by tadjohnston — APPROVED** [review]

**Pull request merged**